pages.pt_BR/osx/head.md

```diff
@@ -0,0 +1,12 @@
+# head
```

Comment:
Mandatory change: this page should be removed with the [original](https://github.com/tldr-pages/tldr/blob/main/pages/osx/head.md) one according our [specification](https://github.com/tldr-pages/tldr/blob/main/CLIENT-SPECIFICATION.md#platform): commands available on more than one platform should be placed in `common/`.

Reply:
Resolved in #9533.